calibration/experiments/gcm_driven_scm/scm_tomls/prognostic_edmfx.toml

```diff
@@ -0,0 +1,26 @@
+[EDMF_min_area]
```

Comment:
No - these tomls just overwrite climaatmos default parameter values but are not calibrated against (but could be 😁). I've changed them before for stability etc... You get an error if the parameters being calibrated are also in here because the yml that's generated doesn't know which to grab.

Reply:
Interesting. I was under the impression that if you don't define the prior the EKP does not read it as a calibrateable parameter.

Reply:
these tomls house CP parameters that need to be overwritten for the given setup, but not calibrated. For instance, `detr_coeff` multiplies a term we want to be 0 for this entrainment formulation. We'll get rid of/calibrate these when the pipeline is updated to ie. the pi group linear entrainment closure. However, I imagine there will often be use cases where you want to set parameters locally but not calibrate them when doing experiments.

Reply:
> Interesting. I was under the impression that if you don't define the prior the EKP does not read it as a calibrateable parameter.

This is still true - I think Julian was referring to first scenario below: 

- If you define a prior for a parameter that's already in the toml, you get an error. 
- If you don't define a prior for a parameter in the toml, it uses the toml value
- If you don't define a prior and the parameter is not in the toml, it uses the ClimaParams default